### PR TITLE
sys-block/zram-init: revbump

### PR DESCRIPTION
**Pull request by beatussum**
Fix ebuild message.

Closes: https://bugs.gentoo.org/852959

### CODE REVIEW

File: sys-block/zram-init/zram-init-8.1-r2.ebuild [hunk 1]
```diff
@@ -1,15 +1,14 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
```
Comment (thesamesam) [inline sys-block/zram-init/zram-init-8.1-r2.ebuild:4]: Please sync the live ebuild too.

Comment (beatussum) [reply sys-block/zram-init/zram-init-8.1-r2.ebuild:4]: It seems to me that the live ebuild is already synced with `=zram-init-11.1` and both have a correct message.

Comment (thesamesam) [reply sys-block/zram-init/zram-init-8.1-r2.ebuild:4]: Oh, sorry, I missed you're changing the old version.. why bother? Just stable something newer (file a bug for that) and cleanup?

Comment (beatussum) [reply sys-block/zram-init/zram-init-8.1-r2.ebuild:4]: So, I close this pull request and file a bug.

**Comment by beatussum** [comment]
https://bugs.gentoo.org/909149

**Pull request closed without merging**